workflows/workflow.go

```diff
@@ -27,3 +29,16 @@ func (workflow *Workflow) Run(interfaceWorkflow InterfaceWorkflow, workflowConte
 
```

Comment:
This is the function which should be handling all the exceptions

Reply:
Done, moved handling the error to workflow. Added print statement in simple workflow to indicate that workflow is aborted.